src/integration-test/java/software/amazon/cloudwatchlogs/emf/MetricsLoggerIntegrationTest.java

```diff
@@ -122,7 +123,7 @@ public void testMultipleFlushOverUDP() throws InterruptedException, InvalidMetri
     private void logMetric(Environment env, String metricName) throws InvalidMetricException {
         MetricsLogger logger = new MetricsLogger(env);
         logger.putDimensions(dimensions);
-        logger.putMetric(metricName, 100, Unit.MILLISECONDS);
+        logger.putMetric(metricName, 100, Unit.MILLISECONDS, StorageResolution.HIGH);
```

Comment:
Why remove it? It's an extra test for us to validate `putMetric` is working as intended

Reply:
Actual test would have been to test the datapoint timestamps with sub second values. In this case it will not work because High Resolution changes are not pushed to canary account region. 

This case was specifically handled in Service Canaries based on the feature flag "EmfHighResolutionMetricsEnabled".